lib/random_data.rb

```diff
@@ -0,0 +1,24 @@
+module RandomData
```

Comment:
Now you're well practiced with writing random data generators, this would be a good time to consider introducing Faker to replace `RandomData`. What's handy about Faker is that its got a lot of commonly used data ready to go: https://github.com/stympy/faker